src/encoded/types/experiment.py

```diff
@@ -142,6 +144,12 @@ class Experiment(Item):
         "files.badges.badge.description",
         "files.badges.badge.badge_icon",
         "files.badges.messages",
```

Comment:
While you're editing the embeds here, you might as well comment out all the files.badges ones for now since we don't have badges for files currently.